src/platform/time-node.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+// High resolution time in micros elapsed since the beginnig of time, which is arbitrary
+// but consistent throughout invocations of the Arcs Runtime instantiation.
+export function now() {
```

Comment:
we already have platform based `now()` methods in date-node.ts and date-web.ts files, which might be a bit confusing. 
maybe worth consolidating them somehow? or at least maybe you could add a comment in the `date-` files?

Reply:
Thanks for letting me know! I've removed the time-* variants!